adapters/p2p2consensus/transaction.go

```diff
@@ -5,32 +5,32 @@ import (
 
 	"github.com/NethermindEth/juno/adapters/p2p2core"
 	"github.com/NethermindEth/juno/core"
-	"github.com/NethermindEth/juno/utils"
 	p2pconsensus "github.com/starknet-io/starknet-p2pspecs/p2p/proto/consensus/consensus"
 )
 
-func AdaptTransaction(t *p2pconsensus.ConsensusTransaction, network *utils.Network) (core.Transaction, core.Class) {
+func AdaptTransaction(t *p2pconsensus.ConsensusTransaction) (core.Transaction, core.Class, error) {
```

Comment:
This is from the previous PR, but why don't we return `consensus.Transaction` instead?